src/screens/NetworkSettings/NetworkSettingsFlowStack.js

```diff
@@ -1,4 +1,5 @@
 import { createStackNavigator } from '@react-navigation/stack';
+import { NetworkPreSettingsNav, NetworkPreSettingsScreen } from './NetworkPreSettingsScreen';
 import { NetworkSettingsDislaimerNav, NetworkSettingsDislaimerScreen } from './NetworkSettingsDisclaimerScreen';
 
 export const NetworkSettingsFlowNav = Symbol('NetworkSettingsFlowStack').toString();
```

Comment:
I know this is not from this PR, but why do we do this? Can't we just use a string directly?

Reply:
Oh! I have 3 motives to use the navigation in this way:
1. Define the "screen navigation name" near the screen component, because the screen name is subject for name changes, and it turns easy to remember change the "screen navigation name"
2. Avoid work with hardcoded names
3. Avoid name collision by using a Symbol, which turn collision impossible, and this feature is good while the application is growing